### PR TITLE
Extralepveto

### PR DESCRIPTION
**Pull request by swozniewski**

### CODE REVIEW

File: physicsobjects.hxx [hunk 1]
```diff
@@ -107,6 +107,31 @@ auto CombineMasks(auto df, const std::string &maskname,
...
+/// false
+///
+/// \param[in] df the input dataframe
+/// \param[out] outputmaskname the name of the new mask to be added as column to
+/// the dataframe \param[in] inputmaskname the name of the input mask \param[in]
+/// ditaupair name of the column of the ditaupair \param[in] index index of the
+/// tau candidate to be ignored by mask
+///
+/// \return a dataframe containing the new mask
+template <class... Masks>
```
Comment (harrypuuter) [inline physicsobjects.hxx:120]: this is not used right ?

File: config/config.py [hunk 2]
```diff
@@ -158,14 +169,14 @@ def build_config():
         "tauES_1prong0pizeroUp",
         {"global": {"tau_ES_shift_DM0": 1.002}},
         [[TauPtCorrection, "global"]],
-        [[LVMu1, "mt"]],
+        [[LVMu1, "mt"], [VetoMuons, "mt"]],
```
Comment (harrypuuter) [inline config/config.py:172]: is this the sanitizing, such that the ES shift is not propagated to the vetoMuons ?

Maybe we should also add the argument names here, like we do for producer instances, to make it easier to understand why this is set here.

**Review by harrypuuter — CHANGES_REQUESTED** [review]
two small things, otherwise very nice :+1:

File: physicsobjects.hxx (hunk 1 above)
Comment (swozniewski) [reply physicsobjects.hxx:120]: oops, indeed. Copy-paste-error

File: config/config.py (hunk 2 above)
Comment (swozniewski) [reply config/config.py:172]: Maybe just for the sanetize_producers argument, which only pops up in some cases? I think repeating all arguments would be too much (e.g. looking at the JEC-shifts-config)

**Review by harrypuuter — APPROVED** [review]

File: config/config.py (hunk 2 above)
Comment (harrypuuter) [reply config/config.py]: yep, fine with me

**Pull request merged**